mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss

```diff
@@ -22,6 +21,16 @@
         .title {
             font-weight: 600;
             flex-grow: 1;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
         }
     }
+
+    .description {
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
```

Comment:
nice, I don't think I had heard of this property before 🚀 https://caniuse.com/css-line-clamp

Reply:
Yeah, its awesome. Browser support is solid.